low-level/framework/sspl_ll_d

```diff
@@ -791,9 +790,10 @@ if __name__ == "__main__":
         sys.exit(os.EX_USAGE)
 
     # Handle signals for log level change
-    signal.signal(signal.SIGUSR1, set_log_level)
+    # signal.signal(signal.SIGUSR1, set_log_level)
     # Handle signal for iem file refresh
     signal.signal(signal.SIGUSR2, refresh_iem_log_file)
+    #DO NOT EDIT: Marker comment to dynamically add signal handler for coverage job
```

Comment:
#DO NOT EDIT: Marker comment to dynamically add signal handler for SIGUSR1 to generate code coverage report